plugins/lang/time.txt

```diff
@@ -37,6 +45,10 @@ TIME_ELEMENT_DAY = dan
 TIME_ELEMENT_DAYS = dani
 TIME_ELEMENT_WEEK = nedelja
 TIME_ELEMENT_WEEKS = nedelje
+TIME_ELEMENT_MONTH = mesec
+TIME_ELEMENT_MONTHS = meseci
+TIME_ELEMENT_IEAR = godina
+TIME_ELEMENT_IEARS = godine
```

Comment:
The same typo as above:
```suggestion
TIME_ELEMENT_YEARS = godine
```

Reply:
done https://github.com/alliedmodders/amxmodx/pull/969/commits/3dbfbb6456420cc3f72bdacdf83fe4b896fb4a24